contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,61 @@
+# See readme.md for instructions on running this code.
+from google import search
+
+def get_google_result(original_content):
+    original_content = original_content.split(' ', 1)[1]
+    try:
+        urls = search(original_content, stop=20)
+    except:
```

Comment:
Please catch only specific errors (I guess you wanted to catch some network errors)